docs/UserGuide.md

```diff
@@ -384,13 +382,14 @@ The simplest way to analyze multiple repos in one go is to use the `--repos` par
 * Format : `java -jar RepoSense.jar --repos REPO_LIST` <br>
 * Example: `java -jar RepoSense.jar --repos https://github.com/reposense/RepoSense.git c:/myRepose/foo/bar` analyzes the two specified repos (one remote, one local) and generates one report containing details of both.
 
-Alternatively, you can use csv config files to customize the analysis as before while specifying multiple repos to analyze.
-* `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
-* `author-config.csv`: Add one row for each author in each repo you want to analyze
+Alternatively, you can use csv config files to further customize the analysis:
```

Comment:
Would this be confusing, since the section heading is about analyzing multiple repos, and not about customizing the analysis? There is already a section on customising analysis via csv files.

Reply:
![Screen Shot 2020-02-04 at 2 28 07 AM](https://user-images.githubusercontent.com/35946746/73679764-03ed3d80-46f6-11ea-8ca8-434c91e57fce.png)

Based on @eugenepeh comment, I made the update.